src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -263,7 +263,13 @@
         "^", SymbolSegment, name="binary_xor", type="binary_operator"
     ),
     LikeOperatorSegment=NamedParser(
-        "like_operator", SymbolSegment, name="like_operator", type="comparison_operator"
+        # TODO: this doesn't feel like the like_operator token is actually used
+        # in the ANSI language nor its descendents (more represented as a
+        # LIKE Pattern Match Operator). Should we delete this segment?
+        "like_operator",
+        SymbolSegment,
+        name="like_operator",
+        type="comparison_operator",
```

Comment:
This was added here: https://github.com/sqlfluff/sqlfluff/commit/446333e628ca19cd07f7fc39517f92f002bc22ba and there's test cases here: https://github.com/sqlfluff/sqlfluff/blob/main/test/fixtures/dialects/ansi/like_operators.sql (along with Jira references to why this was added).

Reply:
I now see the RegEx it's referencing for parsing. I delete the TODO.